hoot-core/src/main/cpp/hoot/core/algorithms/linearreference/WaySublineMatchString.h

```diff
@@ -121,18 +121,7 @@ class WaySublineMatchString
    */
   bool touches(const WaySublineMatchString& other) const;
 
-  QString toString() const
-  {
-    QString result;
-    result += QString("matches:\n");
-    QStringList l;
-    for (size_t i = 0; i < _matches.size(); i++)
-    {
-      l.append(_matches[i].toString());
-    }
-    result += l.join("\n");
-    return result;
-  }
+  QString toString() const;
```

Comment:
Not it :-)  But it was worse, b/c I spent a couple days doing just that (and a few other related things) a couple months ago.  But obviously I didn't get to every header b/c there's still a handful out there like that.  Probably can just fix them as we see them at this point.